src/org/usfirst/frc/team2706/robot/subsystems/Bling.java

```diff
@@ -0,0 +1,130 @@
+package org.usfirst.frc.team2706.robot.subsystems;
+
+import org.usfirst.frc.team2706.robot.commands.bling.BlingController;
+import org.usfirst.frc.team2706.robot.commands.bling.patterns.BlingPattern;
+
+import edu.wpi.first.wpilibj.command.Command;
+import edu.wpi.first.wpilibj.command.Subsystem;
+import edu.wpi.first.wpilibj.networktables.NetworkTable;
+
+public class Bling extends Subsystem{
+    // Bunch of colour presets.
+    public static final long WHITE = 16777215;
+    public static final long YELLOW = 16775680;
+    public static final long GREEN = 524288;
+    public static final long PURPLE = 8388736;
+    public static final long ORANGE = 16753920;
+    public static final long BLUE = 255;
+    public static final long VIOLET = 15631086;
+    public static final long MERGE = 6160762;
+    public static final long TAN = 16767411;
+    public static final long PINK = 14027935;
+    public static final long BLACK = 0;
+    public static final long GOLD = 16766720;
+    public static final long SILVER = 12632256;
+    public static final long RED = 16711680;
+    
+    // All of the pattern numbers
+    public static final String COLOUR_WIPE = "color_Wipe";
+    
+    //  Too far if we're above this.
+    public static final double tooFarDist = 20;
```

Comment:
What are these constants for? It doesn't look like you're using them.

Reply:
Whoops. That's some stuff from last year that merging brought back.